apigee_sync.go

```diff
@@ -160,6 +161,13 @@ func pollChangeAgent() error {
 			return err
 		}
 
+		if changesRequireDDLSync(resp) {
+			log.Info("Detected DDL changes, going to fetch a new snapshot to sync...")
+			return changeServerError{
```

Comment:
Who is catching that error? 

Well if its a valid app state and we have handlers for error - we should not log it as error but use info instead. 

Erro - something unexpected in app.